DependencyInjection/DoctrineExtension.php

```diff
@@ -898,4 +906,34 @@ private function createArrayAdapterCachePool(ContainerBuilder $container, string
 
         return $id;
     }
+
+    private function registerMetadataPhpArrayCacheWarmer(string $entityManagerName, ContainerBuilder $container): void
+    {
+        $cacheWarmerDefinition = $container->register(sprintf('doctrine.orm.%s_metadata_cache.php_array_warmer', $entityManagerName), DoctrineMetadataCacheWarmer::class);
+        $cacheWarmerDefinition->setArguments([
+            new Reference(sprintf('doctrine.orm.%s_entity_manager', $entityManagerName)),
+            $this->getPhpArrayFile($entityManagerName)
+        ]);
+        $cacheWarmerDefinition->addTag('kernel.cache_warmer');
+    }
+
+    private function registerMetadataPhpArrayCache(string $entityManagerName, ContainerBuilder $container): void
```

Comment:
I would personally prefer to not unnecessarily register new service if its only purpose is to inject it to warmer. Service should be instead just inlined, IMHO. Otherwise we are unnecessarily increasing public API surface.

Reply:
`doctrine.orm.default_metadata_cache` is the alias which refers to the cache service. How can I create inline service and refer the alias to it?